core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -98,6 +102,7 @@ class MultiCloudWorkspaceService(userInfo: UserInfo,
       throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.NotImplemented, "MC workspaces are not enabled"))
     }
 
+    createdMultiCloudWorkspaceCounter.inc()
     traceWithParent("createMultiCloudWorkspace", parentSpan)(s1 =>
         createWorkspace(workspaceRequest, s1)
```

Comment:
to only increment the metric on successful workspace creation, you could move the `inc()` into the result of `createWorkspace()`, like so:
```
    traceWithParent("createMultiCloudWorkspace", parentSpan) { s1 =>
      createWorkspace(workspaceRequest, s1) andThen {
        case scala.util.Success(_) => createdMultiCloudWorkspaceCounter.inc()
      }
    }
```
or
```
    traceWithParent("createMultiCloudWorkspace", parentSpan) { s1 =>
      createWorkspace(workspaceRequest, s1) map { ws =>
        createdMultiCloudWorkspaceCounter.inc()
        ws
      }
    }
```